apps/codeforafrica/src/components/ArticleCardList/ArticleCardList.js

```diff
@@ -35,7 +35,7 @@ const ArticleCardList = React.forwardRef(function ArticleCardList(props, ref) {
                   display: { xs: "none", sm: "block" },
                   height: "217.64px",
                 }}
-                src={article.src}
+                src={article?.thumbnail?.src}
```

Comment:
Looking at the designs, it looks to me that the intention is to have the same image used in multiple places i.e. cover image can be used in thumbnail. So we should have something like
```jsx
src={article.thumbnail?.src || article.coverImage?.src}
```
in case thumbnail is not provided.